mods/cnc/mod.yaml

```diff
@@ -154,6 +154,7 @@ ContentInstaller:
 		.: conquer.mix, desert.mix, general.mix, scores.mix, sounds.mix, temperat.mix, winter.mix
 	ShippedSoundtracks: 2
 	MusicPackageMirrorList: http://www.openra.net/packages/cnc-music-mirrors.txt
+	TFDIndexes: 66, 45, 42, 65, 68, 67, 71, 47, 49, 60, 75, 73
```

Comment:
You'll find that the missing mix files are inside MAIN.MIX (RA) and TIBSUN.MIX (TS).